pipeline/jobs/run_forecast_and_output_to_blob.py

```diff
@@ -47,13 +52,23 @@ def main():
         else today.replace(year=today.year + 1, month=1, day=1)
     )
 
+    # Generate forecast. Currently the period is hardcoded to match EDA forecast period.
     forecast_df = run_forecast_with_all_models(
         input_df=input_df,
         model_suggestions=model_suggestions,
         models=models,
-        months_to_forecast=24,
-        start_month=next_month,
+        months_to_forecast=26,
+        start_month=datetime.date(2022, 12, 1),
```

Comment:
Is it by mistake the hardcoded values?

Reply:
Thanks! 
For now, it is hardcoded just for comparison with EDA, because the most recent EDA give 26 months forecast for each combination.
I have documented this in README.md and also how to change this behaviour.
If it is better to use a relative date, I can change it to generate 26 months starting from last month.